### PR TITLE
chore: update wpt

### PR DESCRIPTION
**Pull request by lucacasonato**

### CODE REVIEW

File: tools/wpt/expectation.json [hunk 1]
```diff
@@ -918,8 +918,12 @@
...
         "Equivalence of option values"
       ],
-      "AddEventListenerOptions-signal.any.html": true,
-      "AddEventListenerOptions-signal.any.worker.html": true,
+      "AddEventListenerOptions-signal.any.html": [
+        "Passing null as the signal should throw (listener is also null)"
+      ],
+      "AddEventListenerOptions-signal.any.worker.html": [
+        "Passing null as the signal should throw (listener is also null)"
+      ],
```
Comment (lucacasonato) [inline tools/wpt/expectation.json:926]: https://github.com/denoland/deno/issues/14593

File: tools/wpt/expectation.json [hunk 2]
```diff
@@ -4601,31 +4609,175 @@
...
   "compression": {
-    "compression-bad-chunks.tentative.any.html": true,
-    "compression-bad-chunks.tentative.any.worker.html": true,
-    "compression-including-empty-chunk.tentative.any.html": true,
-    "compression-including-empty-chunk.tentative.any.worker.html": true,
-    "compression-multiple-chunks.tentative.any.html": true,
-    "compression-multiple-chunks.tentative.any.worker.html": true,
-    "compression-output-length.tentative.any.html": true,
-    "compression-output-length.tentative.any.worker.html": true,
+    "compression-bad-chunks.tentative.any.html": [
```
Comment (lucacasonato) [inline tools/wpt/expectation.json:4612]: Tracked in https://github.com/denoland/deno/issues/14591.

File: tools/wpt/expectation.json [hunk 3]
```diff
@@ -3667,6 +3675,7 @@
           "Object Blob object, Blob empty",
           "Object Blob object, Blob NUL",
           "File basic",
+          "Object with a getter that throws",
```
Comment (lucacasonato) [inline tools/wpt/expectation.json:3678]: https://github.com/denoland/deno/issues/14594

**Review by AaronO — APPROVED** [review]
LGTM

**Pull request merged**